src/cmd/builtin/dmesg.c

```diff
@@ -11,16 +11,27 @@
 #include "builtin.h"
 
 static struct optparse_option dmesg_opts[] = {
-    { .name = "clear",  .key = 'C',  .has_arg = 0,
-      .usage = "Clear the ring buffer", },
-    { .name = "read-clear",  .key = 'c',  .has_arg = 0,
-      .usage = "Clear the ring buffer contents after printing", },
-    { .name = "follow",  .key = 'f',  .has_arg = 0,
-      .usage = "Track new entries as are logged", },
+    {
+        .name = "clear",
+        .key = 'C',
+        .has_arg = 0,
+        .usage = "Clear the ring buffer",
+    },
+    {
+        .name = "read-clear",
+        .key = 'c',
+        .has_arg = 0,
+        .usage = "Clear the ring buffer contents after printing",
+    },
+    {
+        .name = "follow",
+        .key = 'f',
+        .has_arg = 0,
+        .usage = "Track new entries as are logged",
+    },
```

Comment:
oh hey, clang seemed to format this struct of options better.  Wonder what subtle difference it is that made clang format this nicer.  Is it the fact field names are used, like ".usage = "?

Reply:
do we want to require tagged structure initialization?  There's a tangible benefit in that code doesn't break when positions change.  OTOH it's more verbose.

Reply:
It is minimally inconsistent in the project, so we should probably pick a way.  It not breaking code if positions change is a certain benefit.  If we're in agreement, I can update RFC7.

Reply:
As an aside, looking through code, clang formatted `flux-exec`'s options in yet another way.

```
     {.name = "noinput",
      .key = 'n',
      .has_arg = 0,
      .usage = "Redirect stdin from /dev/null"},
     {.name = "verbose", .key = 'v', .has_arg = 0, .usage = "Run with more verbosity."},
```

So having the field names seems to be part of clang formatting things better, but it appears to be inconsistent in how it does it.

Reply:
I think we determined during coffee hour that this was due to the "long" `.usage` line and the string break penalty.